.github/workflows/greenplum-abi-tests.yml

```diff
@@ -102,27 +87,35 @@ jobs:
           fetch-depth: 0 # Specify '0' to fetch all history for all branches and tags.
           path: gpdb_src
 
-      - name: Install abi-dumper
+      - name: Install dependencies and abi-dumper
+        run: |
+          sudo gpdb_src/README.ubuntu.bash
+          sudo apt install -y libuv1-dev libpam0g-dev libldap-dev libipc-run-perl abi-dumper
+          sudo ln -fs python2 /usr/bin/python
+
+      - name: Build libxerces 3.1.2
         run: |
-          yum install -y epel-release
-          yum install -y abi-dumper
-          yum install -y libzstd-static
+          wget https://archive.apache.org/dist/xerces/c/3/sources/xerces-c-3.1.2.tar.gz
+          tar -xzf xerces-c-3.1.2.tar.gz
+          pushd xerces-c-3.1.2
+          ./configure
+          make -j`nproc` && sudo make install
+          popd
 
       - name: Build Greenplum
         run: |
           ## TODO: Since abi-dumper requires debug info and it's hard to inject CFLAGS via the script for
           ## releasing Greenplum, we have to manually configure it here. Probably we can improve it in future.
-          export PATH=/opt/python-3.9.13/bin:/opt/python-2.7.18/bin:$PATH
           pushd gpdb_src
           CC='gcc -m64' \
-          CFLAGS='-Og -g3 -Wno-maybe-uninitialized' LDFLAGS='-Wl,--enable-new-dtags -Wl,--export-dynamic' \
-          ./configure --with-quicklz --disable-gpperfmon --with-gssapi --enable-mapreduce --enable-orafce --enable-ic-proxy \
+          CFLAGS='-Og -g3 -gdwarf-4 -Wno-maybe-uninitialized' LDFLAGS='-Wl,--enable-new-dtags -Wl,--export-dynamic' \
```

Comment:
Isn't dwarf-4 the default debug info type? Is this a requirement for ABI checker?

Reply:
On GCC 11 dwarf-5 is default and is not supported by ABI checker, so switching back to dwarf-4.